src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -291,37 +293,47 @@ function positiveTestsStatusText(projection: Projection) {
 
 function hospitalOccupancyStatusText(projection: Projection) {
   const icuUtilization = projection.currentIcuUtilization;
-  const currentlyInICU = projection.currentICUPatients;
+  const currentCovidICUPatients = projection.currentCovidICUPatients;
+  const totalICUCapacity = projection.totalICUCapacity;
+  const nonCovidICUCapacity = Math.floor(projection.nonCovidICUCapacity);
```

Comment:
Please rename this too if you rename the `Projection` property to `nonCovidICUPatients`

Reply:
done